tests/acceptance/features/coreApiShareOperationsToShares1/gettingShares.feature

```diff
@@ -80,7 +80,7 @@ Feature: sharing
       | 2               | 200             |
 
   @smokeTest
-  Scenario Outline: resource cannot be reshared to resource owner
+  Scenario Outline: resource can be reshared to resource owner
```

Comment:
so the behavior changed?

Reply:
test was wrong:
```
  Scenario Outline: resource can be reshared to resource owner
    Given using OCS API version "<ocs_api_version>"
    And group "grp1" has been created
    And user "Carol" has been created with default attributes and without skeleton files
    And user "Carol" has been added to group "grp1"
    And user "Carol" has created folder "/shared"
    And user "Carol" has uploaded file with content "some data" to "/shared/shared_file.txt"
    And user "Carol" has shared folder "/shared" with user "Brian"
    And user "Brian" has accepted share "/shared" offered by user "Carol"
    And user "Brian" has shared folder "/Shares/shared" with group "grp1"
    # no need to accept this share as it is Carol's file
    When user "Carol" gets all the shares shared with her using the sharing API
    Then the OCS status code should be "<ocs_status_code>"
    And the HTTP status code should be "200"
    And the last share id should not be included in the response
    Examples:
      | ocs_api_version | ocs_status_code |
      | 1               | 100             |
      | 2               | 200             |
```
test passed but Carol had `/Shares/shared` in the pending state

how we can reshare to owner: 
1. `user1` shares to `user2`. `user2` reshares to `user1` - 400 `error creating share:error: bad request: jsoncs3: owner/creator and grantee are the same`
2. `user1` shares to `user2`. `user2` reshares to `group1`(`user1` is member) - worked and works now. without accepting-> `user1` has penndig share,  with accepting-> `user1` has accepted share,